webpack/resolve-chunk-plugin/src/ResolveChunkPlugin.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path="./webpackTypes.d.ts" />
```

Comment:
>/// <reference path="./webpackTypes.d.ts" / [](start = 0, length = 43)

can you move this to a folder called `custom-typings` and set the tsconfig `typeRoots` ?

Reply:
That seems to screw up api-extractor.

---
In reply to: [139565750](https://github.com/Microsoft/web-build-tools/pull/375#discussion_r139565750) [](ancestors = 139565750)